### DISCUSSION TITLE
v2

### DISCUSSION THREAD

**Discussion by ahoyahoy** [CodeDredd/pinia-orm#1855] [General]
Hi @CodeDredd,
I'm glad you're back and I hope everything went well! :)

If you can, could you please share with me your plans for the next major version you mentioned? What will be different, a new and time window?

I also wanted to ask if it would be possible to change the ID of the item, for cases where I create an item with a temporary ID and then want to update it with a new ID from the server. So now I have to delete the item and create new ones.

thanks :)
tom

**Comment by CodeDredd** [comment]
Hey Tom :),

so far almost everything is fine now. Thanks for asking and thanks for being back as support again ❤️ .

Well time window is hard to tell. I would say end of year or earlier. Depends if i only want to have the `breaking changes` tickets merged or more.

At the end my big plan is to seperate things more. So the you have ORM as a module and pinia as module. I wanna make this so it's easier also to include other state managment solution behind the ORM layer.

You can't change the id of an item ?
Does this not work?
````ts
useRepo(User).where('id', 123).update({ id: 234 })
````

**Reply by ahoyahoy** [reply]
oh my gosh :D
I never would have thought it could be this easy without a special method. Sorry :)
Thanks, I'll try it soon.
